deepem/data/dataset/zettaset.py

```diff
@@ -46,12 +47,19 @@ def convert_array(arr: ArrayLike) -> np.ndarray:
 
     dset: dict[str, np.ndarray] = {}
 
-    # Image    
+    # Image
+    if zettaset_padding == (0, 0, 0):
+        image_bbox = sample.bbox
+    else:
+        image_bbox = Bbox(
+            sample.bbox.minpt - zettaset_padding, sample.bbox.maxpt + zettaset_padding
```

Comment:
If `zettaset_padding` uses the `zyx` convention, then I think you should convert it to the `xyz` order here?

Reply:
Ah, yeah good catch. I'll update that.